libs/shared/l10n/src/lib/localizer/localizer.base.ts

```diff
@@ -40,7 +40,7 @@ export class LocalizerBase {
   }
 
   protected createBundleGenerator(fetched: Record<string, string>) {
-    async function* generateBundles(currentLocales: string[]) {
+    function* generateBundles(currentLocales: string[]) {
```

Comment:
Any concerns with the `async`ness being out of sync w/ the `packages/fxa-auth-server/lib/l10n
/index.ts` variant now (since the auth-server version is still using an aysnc generator)?

```sh
git grep -Pin -A1 "protected createBundleGenerator" | cat

libs/shared/l10n/src/lib/localizer/localizer.base.ts:42:  protected createBundleGenerator(fetched: Record<string, string>) {
libs/shared/l10n/src/lib/localizer/localizer.base.ts-43-    function* generateBundles(currentLocales: string[]) {
--
packages/fxa-auth-server/lib/l10n/index.ts:51:  protected createBundleGenerator(fetched: Record<string, string>) {
packages/fxa-auth-server/lib/l10n/index.ts-52-    async function* generateBundles(currentLocales: string[]) {
```

https://github.com/mozilla/fxa/blob/586fa0b5781739bb54fbaba54dbf4bcf7aad0a0b/packages/fxa-auth-server/lib/l10n/index.ts#L51-L67

Reply:
I don't think so. The intention is for a follow up ticket to update `fxa-auth-server/lib/l10n` to extend from `LocalizerBase`, so it does need to be compatible. That said, I couldn't see why this needs to be an async generator, so I think it's safe to remove it?

I tested it quickly and removed async from the `auth-server` generator, and TS didn't yell at me at least. So maybe good to go?